src/browser.js

```diff
@@ -130,9 +130,7 @@ function getBrowser (target, dataDir) {
 
     if (target in browsers[process.platform]) {
         const browser = browsers[process.platform][target];
-        return checkBrowserExistsWindows(browser, target).then(function () {
-            return Promise.resolve(browser);
-        });
+        return checkBrowserExistsWindows(browser, target).then(() => Promise.resolve(browser));
```

Comment:
```suggestion
        return checkBrowserExistsWindows(browser, target).then(() => browser);
```

Unnecessary Promise wrapping could be removed. Your call if you want to do it in this PR.